core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -864,6 +865,30 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
     case "REGEXP_MATCH_COUNT":
       unparseRegexMatchCount(writer, call, leftPrec, rightPrec);
       break;
+    case "COT":
+      unparseCot(writer, call, leftPrec, rightPrec);
+      break;
+    case "BITWISE_AND":
+      unparseBitwiseAnd(writer, call.operand(0), call.operand(1), leftPrec, rightPrec);
+      break;
+    case "BITWISE_OR":
+      unparseBiwiseFunctions(writer, call, "|", leftPrec, rightPrec);
+      break;
+    case "BITWISE_XOR":
+      unparseBiwiseFunctions(writer, call, "^", leftPrec, rightPrec);
+      break;
+    case "INT2SHR":
+      unparseInt2shFunctions(writer, call, ">>", leftPrec, rightPrec);
+      break;
+    case "INT2SHL":
+      unparseInt2shFunctions(writer, call, "<<", leftPrec, rightPrec);
```

Comment:
rename unparseInt2shFunctions to unparseIntShiftOperation and revisit all the places.